package.json

```diff
@@ -807,6 +908,62 @@
                     "default": "test/runtests.jl",
                     "description": "A workspace relative path to a Julia file that contains the tests that should be run for live testing.",
                     "scope": "window"
+                },
+                "julia.debuggerDefaultCompiled": {
+                    "type": "array",
+                    "default": [
+                        "Base.",
+                        "-Base.!",
+                        "-Base.all",
+                        "-Base.all!",
```

Comment:
Are these by defining module, or by namespace module?
i.e. will this also compile my overoads of `Base.map(f, ::MyType)`  etc?

Reply:
These are for all methods of that function, so `-Base.map` will force interpreted mode for all overloads.

Reply:
We should look into working out how to do it by definitition location, but that can be a follow up PR?

Reply:
Well, this list could be method based instead of function based, but I'm not sure whether that's particularly useful.